enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -48,6 +54,9 @@ export interface MetabaseColors {
   /** Default background color. */
   background?: string;
 
+  /** Slightly muted background color. */
+  "background-secondary"?: string;
```

Comment:
This follows the convention from `text-secondary`. In the future we can also map `background-tertiary` to `bg-dark` if there is a need.